content/en/docs/Performance/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Performance"
+weight: 150
+date: 2020-11-20
+description: >
+  Baker performance test cases
+---
+
+We report in this page some practical examples of Baker performances we could measure in the
+NextRoll production environment.
+
+Within NextRoll, Baker is often executed on AWS EC2 instances, and thus you find in this page
+many references to
+[EC2 instance types](https://aws.amazon.com/ec2/instance-types/) (`c5.2xlarge`, `c5.2xlarge`, etc.).
+
+### Read from S3 and write to local disk
+
+On an AWS EC2 instance of size `c5.2xlarge`, Baker can read zstandard records from S3, uncompress
+them and apply a basic filtering logic, compressing them back on local files with zstandard at
+compression level 3 and long range mode at 27 using ~90% of capacity of each vCPU (8 in total) and
+~3.5GB of RAM.  
+
+It reads and writes a total of 94 million records in 8'51" (~178k r/w records per second).
```

Comment:
Since we already wrote that baker reads and writes, we should probably just stick to `178k records per second`.
Again, we should round things up a bit in order to make it easily readable IMHO 
```suggestion
It reads and writes a total of 94 million records in less than 9 minutes, that's 178k records per second.
```